charts/alfresco-sync-service/values.yaml

```diff
@@ -105,7 +105,10 @@ postgresql:
   primary:
     extendedConfiguration: |
       shared_buffers = 256MB
-      max_connections = 80
+      max_connections = 100
+      wal_level = minimal
+      max_wal_senders = 0
+      max_replication_slots = 0
```

Comment:
what are those changes unrelated to this PR? 👀

Reply:
these are meant to provide more sensible default config for a "convenience" backend PG and avoid loosing precious connexion slot by reserving them for replication feature that will anyway not be used. This also allow to have a backend that has a configuration that do not requires 16GB RAM like before (with default settings and 400 connections slots allowed)

Reply:
ok but why in a PR that fix a completely different problem and that will hide this change once it get merged to master?

Reply:
Mostly to avoid releasing 2 versions for these 2 different issues (my tests showed the acs-deployment pipeline may be impacted).
Though I'm happy to to split it into 2 PR. But I see little value